src/sign/index.ts

```diff
@@ -56,6 +61,6 @@ export function unsubscribe(email: string) {
   return sign({ email }, UnsubscribeTypes);
 }
 
-export function verifyUnsubscribe(email: string, signature: string) {
-  return verify({ email }, signature, UnsubscribeTypes);
+export function verifyUnsubscribe(email: string, signature: string, signer?: string) {
+  return verify({ email }, signer || wallet.address, signature, UnsubscribeTypes);
```

Comment:
same for unsubscription
```suggestion
  return verify({ email }, signer, signature, UnsubscribeTypes);
```
if you did it for local testing purposes, then probably we need to find another way to test it locally.